### DISCUSSION TITLE
Integrare Imou Cue2 e Imou Ranger2

### DISCUSSION THREAD

**Discussion by michele-diurno** [indomus/forum#3206] [HUB: Home Assistant]
Ciao a tutti,
volevo chiedere se qualcuno è riuscito ad integrare in HA queste telecamere Imou, nello specifico Cue2 e Ranger2.
Ho provato con "Fotocamera generica" senza risultato, poi ho provato con l'integrazione "Imou Life" ma il risultato è stato quello che l'immagine compare ma risulta statica lo streaming non parte, se chiudo e riapro l'immagine statica si aggiorna ma lo streaming resta fermo.
Sempre grazie.

**Comment by LasVeneronas** [comment]
Ciao, si ho integrato senza problemi 3 IMOU camera e l'NVR 9 mesi fa senza grossi problemi. Devi però registrarti su "Open IMOU life" ed ottenere "AppID" e "APPsecret" per integrarlo in Home Assistant. Purtroppo però da circa una settimana ho un errore di configurazione in Home Assistant che non riesco a risolvere dovuto a "Remote API error". Ho provato anche a scaricare nuovamente l'integrazione IMOU ma ottengo errore nella configurazione nel momento in cui inserisco l'URL. 
E' stato aperto un thread dedicato a tale problema 2 settimane fa nella sezione ISSUE, ma non l'hanno ancora risolto. Temo sia avvenutao con l'aggiornamento all'ultima versione di HA.

**Comment by michele-diurno** [comment]
Grazie per le info.
Alla fine ho risolto con il flusso RTSP.
Con il metodo di registrazione su IMOU life purtroppo l'immagine risulta statica, il servizio è passato a pagamento.
Ovvio che con il flusso RTSP le funzioni sono limitate alla sola visione.
rtsp://admin:secretcode@ipdellacam:554/cam/realmonitor?channel=1&subtype=0

secret code lo trovate sotto la cam, ip è quello della camera, la porta mettete quella che vi aggrada di solito è 554, come integrazione ho usato "fotocamera generica"
